### DISCUSSION TITLE
How do you download Video in MKV Format?!

### DISCUSSION THREAD

**Discussion by RyanAhmed2025** [JunkFood02/Seal#973] [Q&A] [answered]
CAPCUT editor, for some reason, doesn't support MP4 Videos as Import, Neither does WhatsApp Status.

I got my answers for how to get MP3 audio, hopefully it'll work but  I really need the MKV format for edits and more. Please, Kindly add this if you are able to.

**Comment by Marco-9456** [comment, marked as answer]
Enable format selection option and select from the listed formats the one that CAPCUT and WhatsApp supports, this is not an issue with seal, rather an issue with the apps you mentioned.

**Reply by RyanAhmed2025** [reply]
Yes. I noticed it too. Capcut works fine but WhatsApp, even though it says it supports MP4, it actually doesn't or randomly does. For over a month, it isn't letting me. This is very frustrating, wish I knew what to do 

Also AV1 downloads are in MKV format.

**Reply by Marco-9456** [reply]
It's not about mp4, sometimes is the video codec and the audio codec is what WhatsApp doesn't support, mp4 is just then container, anyway the next update should have an option which will fix this issue.

**Reply by RyanAhmed2025** [reply]
You mean next Seal Update? Anyways I hope it gets fixed soon. I don't like the Whatsapp video compression.
